proto/vmexec.proto

```diff
@@ -2,6 +2,21 @@ syntax = "proto3";
 
 package vmexec;
 
+import "proto/remote_execution.proto";
+
+message FileSystemLayout {
+  build.bazel.remote.execution.v2.Tree inputs = 1;
+}
+
+message CASFS {
+  // The "remote instance name" to use when interacting with the CAS.
+  string remote_instance_name = 1;
+  // Description of the filesystem layout.
+  FileSystemLayout file_system_layout = 2;
+  // Whether to prepare the filesystem, but skip the execution.
+  bool debug_skip_execute = 3;
```

Comment:
It's a bit confusing to not have `debug_skip_execute` tied to the CASFS configuration because w/o CASFS setup the host could skip sending the `ExecRequest` entirely.

Reply:
I think `debug_skip_execution` makes sense only in the context of CASFS setup because otherwise the host could skip sending the `ExecRequest` entirely.